Detectors/FIT/reconstruction/include/FITReconstruction/CollisionTimeRecoTask.h

```diff
@@ -24,16 +24,18 @@ class CollisionTimeRecoTask : public FairTask
 {
  public:
   CollisionTimeRecoTask();
-  ~CollisionTimeRecoTask() override;
+  virtual ~CollisionTimeRecoTask();
```

Comment:
the correct way is to use `override` as this is overriding a virtual function defined in the base class.